src/masoniteorm/scopes/SoftDeleteScope.py

```diff
@@ -21,6 +25,16 @@ def _with_trashed(self, model, builder):
         builder.remove_global_scope("_where_null", action="select")
         return builder
 
+    def _only_trashed(self, model, builder):
+        builder.remove_global_scope("_where_null", action="select")
+        return builder.where_not_null("deleted_at")
+
+    def _force_delete(self, model, builder):
+        return builder.remove_global_scope(self).set_action("delete")
```

Comment:
@josephmancuso  I am just not sure to understand why we should call `remove_global_scope(self)` before setting action type ? it is to avoid other scope to apply so that `force_delete` really deletes ? If this was not there, the force_delete would be modified to an update of deleted_at with a timestamp through `_query_set_null_on_delete` ?

Reply:
>  I am just not sure to understand why we should call remove_global_scope(self) before setting action type

doesn't matter which order we do it in. 

```python
builder.set_action("delete").remove_global_scope(self)
```

Works just as well. We do have to remove the scopes because yeah we don't want other scopes changing the action type back to update. We might not even have to set the action here, just removing the scope might actually work ..